alert-database/src/main/resources/liquibase/6.4.0/jira-custom-fields.xml

```diff
@@ -76,4 +76,33 @@
         />
     </changeSet>
 
+    <changeSet author="jrichard" id="add-defined-field-field-mapping">
```

Comment:
Do we still need these fields in the DB if we aren't storing them with these tables? Does the UI config adequately handle the current dynamic field generation without these being stored?

Reply:
I added these, because our current job config handling strips out all non-registered fields from the key value pairs. When we change the job handling, this will no longer be necessary.